front/lib/api/assistant/jit_actions.ts

```diff
@@ -142,15 +216,12 @@ async function getJITActions(
 
       // Check tables for the table query action.
       const filesUsableAsTableQuery = filesUsableForJIT.filter(
-        (f) => f.contentType === "text/csv" // TODO: there should not be a hardcoded value here
+        (f) => f.isQueryable
       );
 
       if (filesUsableAsTableQuery.length > 0) {
-        // TODO(JIT) Shall we look for an existing table query action and update it instead of
-        // creating a new one? This would allow join between the tables.
+        // TODO(jit) Shall we look for an existing table query action and update it instead of creating a new one? This would allow join between the tables.
         const action: TablesQueryConfigurationType = {
-          // The description here is the description of the data, a meta description of the action
```

Comment:
You're dropping this comment unexpectedly I think?

Reply:
Yep (merge conflict)